Ubuntu: https://bugs.launchpad.net/ubuntu/+source/glibc/+bug/302087
Upstream: http://sourceware.org/bugzilla/show_bug.cgi?id=9720
Description: fixes resource.h to compile without -D_GNU_SOURCE
http://sourceware.org/cgi-bin/cvsweb.cgi/libc/sysdeps/unix/sysv/linux/bits/resource.h.diff?cvsroot=glibc&r1=1.20&r2=1.21
http://sourceware.org/cgi-bin/cvsweb.cgi/libc/sysdeps/unix/sysv/linux/sparc/bits/resource.h.diff?cvsroot=glibc&r1=1.10&r2=1.11

===================================================================
RCS file: /cvs/glibc/libc/sysdeps/unix/sysv/linux/bits/resource.h,v
retrieving revision 1.20
retrieving revision 1.21
diff -u -r1.20 -r1.21
--- libc/sysdeps/unix/sysv/linux/bits/resource.h	2008/05/01 05:37:45	1.20
+++ libc/sysdeps/unix/sysv/linux/bits/resource.h	2009/01/08 10:16:12	1.21
@@ -158,10 +158,11 @@
 #define RUSAGE_SELF RUSAGE_SELF
 
   /* All of its terminated child processes.  */
-  RUSAGE_CHILDREN = -1,
+  RUSAGE_CHILDREN = -1
 #define RUSAGE_CHILDREN RUSAGE_CHILDREN
 
 #ifdef __USE_GNU
+  ,
   /* The calling thread.  */
   RUSAGE_THREAD = 1
 # define RUSAGE_THREAD RUSAGE_THREAD
===================================================================
RCS file: /cvs/glibc/libc/sysdeps/unix/sysv/linux/sparc/bits/resource.h,v
retrieving revision 1.10
retrieving revision 1.11
diff -u -r1.10 -r1.11
--- libc/sysdeps/unix/sysv/linux/sparc/bits/resource.h	2008/05/01 05:37:45	1.10
+++ libc/sysdeps/unix/sysv/linux/sparc/bits/resource.h	2009/01/08 10:16:12	1.11
@@ -174,10 +174,11 @@
 #define RUSAGE_SELF RUSAGE_SELF
 
   /* All of its terminated child processes.  */
-  RUSAGE_CHILDREN = -1,
+  RUSAGE_CHILDREN = -1
 #define RUSAGE_CHILDREN RUSAGE_CHILDREN
 
 #ifdef __USE_GNU
+  ,
   /* The calling thread.  */
   RUSAGE_THREAD = 1
 # define RUSAGE_THREAD RUSAGE_THREAD
